### PR TITLE
Added create_recurring function in payment

### PR DESCRIPTION
**Pull request by somit**
Added code in payments for creating a recurring payment using token  as per 

https://docs.razorpay.com/v1/page/recurring-payments-cug-e-mandate-version#paymentscreaterecurring

### CODE REVIEW

**Comment by somit** [comment]
Any updates on the pull request..

File: tests/test_client_payment.py [hunk 1]
```diff
@@ -92,3 +92,14 @@ def test_bank_transfer_fetch(self):
         response = self.client.payment.bank_transfer(self.payment_id)
         self.assertEqual(response['virtual_account_id'], 'va_8J2ny4Naokqbpe')
         self.assertEqual(response['payment_id'], self.payment_id)
+
+
```
Comment (NyanKiyoshi) [inline tests/test_client_payment.py]: Too many blank lines: should be one.

File: razorpay/resources/payment.py [hunk 2]
```diff
@@ -103,3 +103,14 @@ def bank_transfer(self, payment_id, data={}, **kwargs):
         """
         url = "{}/{}/bank_transfer".format(self.base_url, payment_id)
         return self.get_url(url, data, **kwargs)
+
+
```
Comment (NyanKiyoshi) [inline razorpay/resources/payment.py]: Too many blank lines: should be one.

File: tests/test_client_payment.py (hunk 1 above)
Comment (somit) [reply tests/test_client_payment.py]: Done

**Comment by somit** [comment]
We are using these changes in production. Kindly merge it so that we can directly do a pip install.

**Comment by Goutam192002** [comment]
How soon can this pull request be merged?

**Comment by kkyusufk** [comment]
@somit We already have this code added here
https://github.com/razorpay/razorpay-python/blob/master/razorpay/resources/payment.py#L218

**Comment by kkyusufk** [comment]
@somit We apologize for the delay in response, I realized this PR is very old and is stale.
Please do not hesitate to open new PR's in the future and we will surely get it merged.
Thank you for the PR.

**Comment by kkyusufk** [comment]
Closing as this functionality is already present

**Pull request closed without merging**